server.js

```
@@ -0,0 +1,221 @@
+                fs.writeFile(`./public/${parsedInfo.elementName}.html`, template, (err) => {
+                    if (err) {
+                        return console.log(err);
+                    }
+                    console.log('Filed Saved!');
+                })
+                res.writeHead(200, {
+                    'Content-Type': 'application/json'
+                });
+                let newFiles = []
+                fs.readdir('./public/', (err, files) => {
+                    for (let i = 0; i < files.length; i++) {
+                        if (files[i] !== '.keep' && files[i] !== '404.html' && files[i] !== 'css' && files[i] !== 'index.html' && files[i] !== 'styles.css') {
+                            newFiles.push(files[i]);
+                        };
+                    };
+                    let newIndex = `<!DOCTYPE html>
+                    <html lang="en">
+                    
+                    <head>
+                        <meta charset="UTF-8">
+                        <title>The Elements</title>
+                        <link rel="stylesheet" href="/styles.css">
+                    </head>
+                    
+                    <body>
+                        <h1>The Elements</h1>
+                        <h2>These are all the known elements.</h2>
+                        <h3>These are ${newFiles.length} of them.</h3>
+                        <ol>`;
+                    for (let i = 0; i < newFiles.length; i++) {
+                        let newList = `<li>
+                        <a href="/${newFiles[i]}">${newFiles[i].split('.')[0]}</a>
+                    </li>`;
+                        newIndex += newList;
+                    };
+                    let htmlEnd = `    </ol>
+                    </body>
+                    
+                    </html>`;
+                    fs.writeFile('./public/index.html', newIndex + htmlEnd, (err) => {
+                        if (err) {
+                            throw err;
```

This should return an error to the client using either a 400/500 error